android/src/main/java/org/wonday/pdf/PdfView.java

```diff
@@ -362,4 +375,29 @@ private Uri getURI(final String uri) {
         }
         return parsed;
     }
+
+    private void setTouchesEnabled(final boolean enabled) {
+        setTouchesEnabled(this, enabled);
+    }
+
+    private static void setTouchesEnabled(View v, final boolean enabled) {
+        if (enabled) {
+            v.setOnTouchListener(null);
+        } else {
+            v.setOnTouchListener(new View.OnTouchListener() {
+                @Override
+                public boolean onTouch(View v, MotionEvent event) {
+                    return true;
+                }
+            });
+        }
+
+        if (v instanceof ViewGroup) {
+            ViewGroup vg = (ViewGroup) v;
+            for (int i = 0; i < vg.getChildCount(); i++) {
+                View child = vg.getChildAt(i);
+                setTouchesEnabled(child, enabled);
```

Comment:
To confirm... setTouchesEnabled is to allow touch on the thumbnail opens up the PDF viewer correct?

Reply:
If I don't do this, then the app crashes as the PDFView tries to look for links on a page that been discarded since we left the screen.

Reply:
Please merge ahead if correct.

Reply:
Will leave unmerged for now, since I don't want to merge to master until PR is approved by react-native-pdf maintainers.

Reply:
ic